projects/tinkerbell/cluster-api-provider-tinkerbell/patches/0001-Update-tinkerbell-tink-version.patch

```diff
@@ -0,0 +1,2318 @@
+From 4020b9cfa6a4bb320a50342f061e2d7e3a9636d8 Mon Sep 17 00:00:00 2001
```

Comment:
I can't quite work out why this patch was needed; could you elaborate?

Reply:
This version of tink has the vlan id in the hardware spec. In my manual testing, I found that without it when CAPT patches tink hardware objects the vlan id field is always removed.